### ISSUE TITLE
VxCentralScan: support latest Fujitsu

### ISSUE THREAD

**Issue by carolinemodic** [votingworks/vxsuite#1741]
The fujitsu-7160 is being end of life'd. Fujitsu is sending us the 8170, we should test it out and make sure everything works as expected and it is safe to upgrade to. 

https://votingworks.slack.com/archives/CFSJEQ26N/p1650991391990949

**Comment by mattroe** [comment]
fi-8170 does not work out of the box. Recognized by VxCentralScan, but doesn't scan and creates empty batches.

We should ensure we have driver support for the fi-8170 & fi-8190 as well as confirm we have driver support for fi-7800 and fi-7900.

**Comment by arsalansufi** [comment]
We've successfully used the fi-8170 to scan ballots on VxCentralScan, both on a Debian 11 Mac VM and on a Debian 11 VxDev ThinkPad.

To get thinks working with Debian 11, we had to prepare and install a backport of the latest `libsane` package. This package is included in Debian 12, so things should work out of the box once we upgrade to Debian 12. In the meantime, while we're on Debian 11, we have both an arm64 and amd64 backport. The backports are in the VotingWorks S3, in the `debian-backports` bucket:

- `libsane-backport-arm64.zip` (for Mac VMs)
- `libsane-backport-amd64.zip` (for real hardware)

The following sequence of commands should do the trick:

```
unzip libsane-backport-[arm64|amd64].zip
cd libsane-backport-[arm64|amd64]
sudo apt install ./libsane-common_1.2.1-4~bpo11+1_all.deb ./libsane1_1.2.1-4~bpo11+1_amd64.deb ./libsane1-dbgsym_1.2.1-4~bpo11+1_amd64.deb ./libsane-dev_1.2.1-4~bpo11+1_amd64.deb ./sane-utils_1.2.1-4~bpo11+1_amd64.deb ./sane-utils-dbgsym_1.2.1-4~bpo11+1_amd64.deb
```
